### DISCUSSION TITLE
没有绑定 Telegram的按钮

### DISCUSSION THREAD

**Discussion by ChaneyChokin** [BurnHair/docs#161] [技术支持]
![QQ截图20240715084713](https://github.com/user-attachments/assets/2f6c1c26-621e-4e00-a5db-ab2c33b9835a)
在设置页面没有找到绑定 Telegram的按钮。

**Comment by BennyThink** [comment]
可能是你的网络问题，再刷新一下试试看呢

**Reply by lyp82nlf** [reply]
@BennyThink 我也遇到了，没有绑定的按钮

**Reply by ChaneyChokin** [reply]
> 可能是你的网络问题，再刷新一下试试看呢

我切换了许多主流国家/地区的代理节点测试（并且清除过浏览器缓存），仍然没有绑定按钮，我个人判断不是网络原因造成的。

**Reply by BennyThink** [reply]
能打开 F12看下有没有报错之类的吗？console和network这两个标签页

**Reply by lyp82nlf** [reply]
> 能打开 F12看下有没有报错之类的吗？console和network这两个标签页

![image](https://github.com/user-attachments/assets/45264ce9-f815-41dc-9eff-e79f6081bac1)
这个获取不到

**Reply by ChaneyChokin** [reply]
> 能打开 F12看下有没有报错之类的吗？console和network这两个标签页

抱歉，已经解决了。是浏览器的代理拓展插件SwitchyOmega导致的，关闭该插件或全局代理可出现绑定按钮，但规则内添加不会出现按钮，猜测是该插件的代理规则有问题。
